src/core/lv_obj_style.c

```diff
@@ -224,7 +224,7 @@ void lv_obj_enable_style_refresh(bool en)
 
 lv_style_value_t lv_obj_get_style_prop(const lv_obj_t * obj, lv_part_t part, lv_style_prop_t prop)
 {
-    lv_style_value_t value_act;
+    lv_style_value_t value_act = { .num = 0 };
```

Comment:
```suggestion
    lv_style_value_t value_act = { .ptr = NULL };
```

On 64 bit machines `num` is `int32_t` (4 bytes) but `ptr` is 8 bytes.